openid-4-verifiable-presentations-1_0.md

```diff
@@ -261,6 +261,15 @@ This specification defines the following new parameters:
 
 A public key to be used by the Wallet as an input to the key agreement to encrypt Authorization Response (see (#jarm)). It MAY be passed by the Verifier using the `jwks` or the `jwks_uri` claim within the `client_metadata` or `client_metadata_uri` request parameter.
```

Comment:
```suggestion
The Wallet MAY use a Verifier's public key to encrypt the Authorization Response (see (#jarm)). The Verifier MAY provide this public key using the jwks or jwks_uri claim within the client_metadata or client_metadata_uri request parameter. Alternatively, other methods like OpenID Federation can be used, where the Verifier's metadata is provided within a Federation Trust Chain.
```

Reply:
I'll provide a PR with the ref to Federation as well.

Reply:
this part of the spec is not subject of this PR